cypress/integration/check-eligibility.spec.js

```diff
@@ -62,6 +63,48 @@ describe('Create New Audit', () => {
     });
   });
 
+  describe('Eligibility validation via API', () => {
+    it('should return eligibility errors from the remote server', () => {
+      cy.intercept(
+        {
+          method: 'POST',
+          url: '/sac/eligibility',
+        },
+        {
+          elegible: false,
+          errors: 'Not elegible.',
```

Comment:
Very small thing, but please change "elegible" to "eligible" throughout this spec.

Reply:
Will do on the elegible to eligible. As for the update to the skipped spec, I accidentally updated it and then reverted it in my last commit.

Reply:
The error that gets thrown is from the Blank Form test block, specifically the 'will not submit' test. It is not a Cypress error but an error specified in our test script. I wonder if I change my new tests from a click on the Continue button to a form submit with the e.preventDefault added to the commands.